src/components/countdown/__tests__/countdown-test.js

```diff
@@ -0,0 +1,70 @@
+var countdown = require('../code/lib/countdown.js');
+var moment = require('moment');
+
+var dateFormat = 'YYYY-MM-DD';
+
+describe('until', function() {
+  describe('future dates', function() {
+    beforeEach(function() {
+      var futureSeconds = 999999;
+      this.date = moment().add(futureSeconds, 'seconds').format(dateFormat);
```

Comment:
It would be easier to make ```this.date``` a local variable as ```this``` makes it unclear what the scope of the variable is